Users/Install.rst

```diff
@@ -199,7 +199,7 @@ application also asks for the packages needed in case it does not
 have it.
 
 The requirements files ( Gemfile, requirements.txt, etc.) are in
-the coala-bears repository and you should not get them from 
+the coala-bears repository and you should not get them from
```

Comment:
how did this even happen; we have SpaceConsistencyBear enabled in the .coafile .